oemoflex/preprocessing.py

```diff
@@ -186,7 +225,7 @@ def get_parameter_values(scalars_df, parameter_name):
 def update_shortage(data_preprocessed_path, scalars):
     logging.info("Updating shortage file")
 
-    shortage_file = os.path.join(data_preprocessed_path, 'elements', 'shortage.csv')
+    shortage_file = os.path.join(data_preprocessed_path, 'elements', 'electricity-shortage.csv')
```

Comment:
Have you thought about not to repeat theses file names as since there are already defined in `components.csv`?

Reply:
Good idea! We can do this as part of the cleaning up of the information flows. I opened a new project that collects the issues involved here. https://github.com/modex-flexmex/oemo-flex/projects/1 

You can either add it to an existing issue or open a new one if that fits better.